app/upgrades/v16/upgrades.go

```diff
@@ -195,6 +195,58 @@ func CreateUpgradeHandler(
+5BBG.                                     5BBB!      5BBB?  JBBB~      !BBBP.    !BBBB^            :GBBB!                                    .GBB5
+5BBG.                                    .PBBB^      :GBBG:^BBB7       ~BBBG.    .PBBB5.           ?BBBB!                                    .GBB5
+5BBG.                                    :GBBG:       !BBBY5BBY        ^BBBB^     ^PBBBP~        :JBBBBB!                                    .GBB5
+5BBG.                                    ^BBBG.        JBBBBBP.        :GBBB~      :JGBBBPJ7!!7?5BBJGBBB!                                    .GBB5
+5BBG.                                    !#B#P         .P#BBG^         .PBB#7        :7YPBBBBBBG57:.GBBB~                                    .GBB5
+5BBG.                                    !5Y5?          ^JJJ!           ?5Y5!           .:^^^^:.   ^BBBB:                                    .GBB5
+5BBG.                                                                                              YBBBY                                     .GBB5
+5BBG.                                                                               .:           ^YBBBP:                                     .GBB5
+5BBG.                                                                              .5BPY?7!!!!7JPB#BGJ.                                      .GBB5
+5BBG.                                                                              :J5GBBB#BBBBBGPJ!:                                        .GBB5
+5BBG.                                                                                                                                        .GBB5
+5BBG.                                                                                                                                        .GBB5
+5BBG.                                                                                                                                        .GBB5
+5BBG.                                                                                                                                        .GBB5
+JBBB^                                                                                                                                        ^BBBJ
+:GBBP^                                                                                                                                      ^PBBG:
+^PBBBJ^.                                                                                                                                .^JBBBP^
+.7PB#BG5J????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????J5GB#BP7.
+    .~?5GBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBG5?~.
+        .::^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^::.
+`)
+
```

Comment:
lol, this is hardcoded. I thought it was some magic function that generated it

Reply:
We should make determistic cosmwasm contract that generates ascii arts